src/components/SearchBar.js

```diff
@@ -41,7 +43,7 @@ export default class SearchBar extends Component {
 			$('.typeahead-selector').show();
 			this.setState({query: event.target.value});
 			console.log("haetaan");
-			$.get('http://www.omdbapi.com/?s=' + event.target.value, (result) => {
+			$.get('http://www.omdbapi.com/?apikey' + apiKey + '&s=' + event.target.value, (result) => {
```

Comment:
This query is not working if you check network tab in chrome. Compare this string to others 👍